src/pharmpy/tools/structsearch/tool.py

```diff
@@ -99,31 +99,38 @@ def run_tmdd(context, model):
 
 
 def run_pkpd(context, model):
-    wf = create_fit_workflow(model)
+    pk_dataset = model.dataset
+    pk_dataset = pk_dataset[pk_dataset["DVID"] != 2]
+    pk_model = model.replace(
+        dataset=pk_dataset,
+        description=model.description + ". Removed rows with DVID=2.",
+        name="PK_" + model.name,
+    )
+
+    wf = create_fit_workflow(pk_model)
     task_results = Task('results1', bundle_results)
     wf.add_task(task_results, predecessors=wf.output_tasks)
-    pk_model = call_workflow(wf, 'results_pd', context)
+    pk_model_fit = call_workflow(wf, 'results_pd', context)
```

Comment:
I was thinking it might be a bit clearer to index out the model object from the tuple here? I.e.: `pk_model_fit = call_workflow(wf, 'results_pd', context)[0]` and then `pk_model_fit` in the line below.

Reply:
Leaving it as tuple since then the tuples can be added together later when writing `pk_model_fit + pkpd_models_fit`.